inst/app/server.r

```diff
@@ -44,23 +39,19 @@ shiny::shinyServer(
         return("Versjon 0.4.0")
       }
     })
-
     output$plot_kvalitet <- shiny::renderUI({
       if (length(input$valgtBo) == 0) {
         return(NULL)
       } else {
         plotly::plotlyOutput("plotly_plot")
       }
     })
-
     output$plot_variasjon <- shiny::renderUI({
       plotly::plotlyOutput("heatmap", width = "auto", height = "800px")
     })
-
     output$heatmap <- plotly::renderPlotly({
-      SKDEresultater::create_heatmap(data = variasjon_data())
+      SKDEresultater::create_heatmap(data = variasjon_data(), type = "heatmaply")
```

Comment:
Jeg ville droppet å sende `type = "heatmaply"` og heller bruke `"heatmaply"` som default i `create_heatmap`-funksjonen. Så dropper du alle endringene i `server.r`.

Reply:
your wish is my command